src/components/PlantsV2/components/PlantingSiteDensityCard.tsx

```diff
@@ -0,0 +1,93 @@
+import OverviewItemCard from 'src/components/common/OverviewItemCard';
+import { Box, Typography, useTheme } from '@mui/material';
+import strings from 'src/strings';
+import React from 'react';
+import { useAppSelector } from 'src/redux/store';
+import { selectPlantingSite } from 'src/redux/features/tracking/trackingSelectors';
+import { ObservationResults } from 'src/types/Observations';
+import { getShortDate } from 'src/utils/dateFormatter';
+import { useLocalization } from 'src/providers';
+import ProgressChart from 'src/components/common/Chart/ProgressChart';
+import { selectPlantingsDateRange } from 'src/redux/features/Plantings/plantingsSelectors';
+
+type PlantingSiteDensityCardProps = {
+  plantingSiteId: number;
+  observation?: ObservationResults;
+};
+
+export default function PlantingSiteDensityCard({
+  plantingSiteId,
+  observation,
+}: PlantingSiteDensityCardProps): JSX.Element {
+  const theme = useTheme();
+  const plantingSite = useAppSelector((state) => selectPlantingSite(state, plantingSiteId));
+  const plantingsSinceObservation = useAppSelector((state) =>
+    selectPlantingsDateRange(state, observation?.completedDate ? [observation?.completedDate] : [])
+  );
+  const locale = useLocalization();
+
+  const weightedSum =
+    plantingSite?.plantingZones?.map((z) => z.targetPlantingDensity * z.areaHa)?.reduce((a, b) => a + b) ?? 0;
+  const targetPlantingDensity = plantingSite?.areaHa ? weightedSum / plantingSite.areaHa : 0;
```

Comment:
do we need to worry about divide-by-zero? (areaHa should never by 0 but it is a number type and theoretically can)

Reply:
if `plantingSite?.areaHa` is zero then it should take the else branch, due to 0 being falsey 😄

Reply:
ugh ofc.. 🤦